src/Components/TestimonialCard.js

```diff
@@ -1,5 +1,35 @@
 import React from "react";
+import styles from "../Css/Testimonial.module.css";
 
-export default function TestimonialCard() {
-  return <></>;
+const styleImage = {
+  position: "absolute",
+  width: "100px",
+  transfrom: "translateX(-50%)"
+};
+
+const bg_color = {
+  backgroundColor: "#008dc8",
+  textAlign: "center"
```

Comment:
> [SUGGESTION]

Ideally we should avoid inline styling. Try if you could move this to `.css` file. Later, it will be problem when you want to scale or reuse your components as the website grows.